mkosi/__init__.py

```diff
@@ -2817,6 +2817,9 @@ def install_debian_or_ubuntu(args: MkosiArgs, root: Path, *, do_run_build_script
         for kver, kimg in gen_kernel_images(args, root):
             run_workspace_command(args, root, ["kernel-install", "add", kver, Path("/") / kimg])
 
+    root.joinpath("etc/locale.conf").write_text("LANG=C.UTF-8\n")
```

Comment:
Before the 2.35 release, C.UTF-8 has always been backported by distros (except Arch) so I'm pretty sure we can rely on it being there on Debian.

Only on Arch it has to be generated (which reminds me that I should fix that since we don't use C.UTF-8 on Arch atm).